src/Bundle/Resources/config/services.xml

```diff
@@ -86,5 +86,27 @@
         >
             <argument type="tagged" tag="sylius_mailer.email_modifier" />
         </service>
+
+        <service id="Sylius\Bundle\MailerBundle\Cli\Dumper\EmailDetailsDumper">
```

Comment:
Following @mbabker advices, I'm not sure if it'd be better to follow Symfony best practices for service ids instead.